samples/subsys/canbus/isotp/README.rst

```diff
@@ -0,0 +1,59 @@
+.. _isotp-sample:
+
+ISO-TP library
+##############
+
+Overview
+********
+
+This sample demonstrates how to use the ISO-TP library.
+Messages are exchanged between two boards. A long message, that is sent with
+a block-size (BS) of eight frames, and a short one that has an minimal
+separation-time (STmin) of five milliseconds.
+The send function call for the short message is none-blocking, and the send
+function call for the long message is blocking.
+
+Building and Running
+********************
+
+
+Integrated CAN controller
+=========================
+
+For the NXP TWR-KE18F board:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/CAN
+   :board: twr_ke18f
+   :goals: build flash
+
+Requirements
+************
+
+* Two boards with CAN bus support, connected together
+
+Building and Running
+********************
+
```

Comment:
The `Building and Running` is here twice, probably the first one is enough.

Reply:
Copy and Paste fail.
Removed.